configure.ac

```diff
@@ -26,12 +64,29 @@ PKG_CHECK_MODULES([SQLITE3], [sqlite3],
   [SQLITE3_CFLAGS+=" -DSQLITE_ENABLE_SESSION -DSQLITE_ENABLE_PREUPDATE_HOOK"])
 PKG_CHECK_MODULES([ZMQ], [libzmq])
 
+# When cross-compile to windows, we don't need mover folder
+# also we need to pass extra CPP flag for jsonrpccpp library
+# CONDITIONAL START
+  
+
+
+AM_COND_IF( [WINDOWS], [
+CXXFLAGS+=" -DGLOG_NO_ABBREVIATED_SEVERITIES"
```

Comment:
This is something that we should add generally, not conditioned on Windows.  While this solves a problem particular to Windows, the code does not need the abbreviated severities anyway, so we should disable them in general.  So just add this to `CXXFLAGS` unconditionally.

Reply:
Ok!